checked/bytes.go

```diff
@@ -54,6 +54,7 @@ func NewBytes(value []byte, opts BytesOptions) Bytes {
 		value: value,
 	}
 	b.SetFinalizer(b)
+	b.TrackObject(b.value)
```

Comment:
Yes, it's a whole-program flag. If one knew though that there are leaks happening during a certain operation in a certain package, one could use these functions to enable instrumentation only when suspicious code is being executed.